TLM/TMPE.API/Traffic/Data/ExtSegment.cs

```diff
@@ -27,12 +27,18 @@ public struct ExtSegment : IEquatable<ExtSegment> {
         /// </summary>
         public bool buslane;
 
+        /// <summary>
+        /// The Lane IDs as an array for fast lookup by index.
+        /// </summary>
+        public uint[] lanes;
```

Comment:
Is this supposed to be in the API?

Reply:
I don't want it to be that way, but that's how `ExtSegmentManager` is written. As much as I'd like to secure my reputation as the API killer, I'd rather not change that unless forced to as I basically was in the traffic light stuff.

Reply:
It's not really an API. It's exactly the same mess as the traffic light stuff.

@krzychu124 do you have an opinion on this?  The nice thing in this case is, it's a much smaller problem, and easier to define. than the traffic light stuff. If we did want to clean it up, we could make sure there's new API in place to do whatever people might have needed with it.